chat-plugins/room-events.js

```diff
@@ -46,12 +47,18 @@ exports.commands = {
 			const eventId = toId(eventName);
 			if (!eventId) return this.errorReply("Event names must contain at least one alphanumerical character.");
 
+			if (room.events[eventId] && this.cmd === 'add') {
```

Comment:
Do the other way around, too: `edit` shouldn't work if the room event doesn't already exist